dvol_python/test_dvol.py

```diff
@@ -126,6 +161,7 @@ def test_list_multi_volumes(self, volumes):
             sorted([line.split() for line in rest]),
         )
 
+    @settings(max_examples=5)
```

Comment:
Rather than doing this here, which implies that the `max_examples` is somehow intrinsic to the test, instead put something like this in the `tests/__init__.py` file:

```python
import os
from hypothesis import settings

settings.register_profile("ci", settings(max_examples=1000))
settings.register_profile("dev", settings(max_examples=5))
settings.register_profile("debug", settings(max_examples=10, verbosity=Verbosity.verbose))

settings.load_profile(os.environ.get(u'HYPOTHESIS_PROFILE', 'default'))
```

And then on your laptop (perhaps in an `.envrc`?) set `HYPOTHESIS_PROFILE=dev`.

This might seem like gold-plating, but if we're sticking with Python tests, it'll be worth it.